WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductSelectorViewModel.swift

```diff
@@ -172,16 +172,35 @@ final class ProductSelectorViewModel: ObservableObject {
         configureProductSearch()
     }
 
+    private func debug_helper(product: Product) {
+        print("🍍ProductID: \(product.productID) selected. \(product.name)")
+        print("🍍initialSelectedItems \(initialSelectedItems)")
+        print("🍍selectedProductIDs \(selectedProductIDs)")
+        print("🍍selectedProductVariationIDs \(selectedProductVariationIDs)")
+    }
+
     /// Select a product to add to the order
     ///
     func selectProduct(_ productID: Int64) {
         guard let selectedProduct = products.first(where: { $0.productID == productID }) else {
             return
         }
-        if let onProductSelected = onProductSelected {
-            onProductSelected(selectedProduct)
+
+        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.productMultiSelectionM1) {
```

Comment:
I believe you should move this check to Order creation code and add a config for the `ProductSelectorViewModel` instead since this view is reused in different projects.

Reply:
I see! Thanks! I moved it to the `EditableOrderViewModel` instead, and injected the flag into the `ProductSelectorViewModel` on https://github.com/woocommerce/woocommerce-ios/pull/8935/commits/12a1d4ba6490a925c4b857598a3e9338f47fb6a7